Dockerfile

```diff
@@ -19,6 +19,11 @@ RUN yum -y install epel-release \
     tango-db \
     tango-accesscontrol \
     tango-test \
+    python-pytango \
+    gcc \
```

Comment:
`pip install pymodbys` didn't required that, so i thought  only `gcc` will be enough. The other thing is  that we want the image to be as small as possible.